sw/device/silicon_creator/lib/drivers/keymgr_functest.c

```diff
@@ -7,11 +7,10 @@
 
 #include "sw/device/lib/arch/device.h"
 #include "sw/device/lib/base/macros.h"
-#include "sw/device/lib/dif/dif_aon_timer.h"
 #include "sw/device/lib/dif/dif_flash_ctrl.h"
 #include "sw/device/lib/dif/dif_kmac.h"
 #include "sw/device/lib/dif/dif_otp_ctrl.h"
-#include "sw/device/lib/dif/dif_pwrmgr.h"
+#include "sw/device/lib/dif/dif_rstmgr.h"
```

Comment:
Should also include `sw/device/lib/testing/otp_ctrl_testutils.h`

Reply:
Done.